source/api/commands/route.md

```diff
@@ -461,16 +437,10 @@ The following XHR's which were `xhr.open(...)` with these URLs would:
 
 ***Requests that don't match any routes***
 
-You can force routes that do not match a route to return `404`:
-
-Status | Body | Headers
---- | --- | ---
-`404` | "" | `null`
-
-If you'd like to enable this behavior you need to pass:
+You can force requests that do _not_ match a route to return a `404` status and an empty body by passing an option to the `cy.server()` like so:
 
 ```javascript
-cy.server({force404: true})
+cy.server({ force404: true })
```

Comment:
Again, I'm directing the reader to the "canonical" documentation about the `force404` option on the `server` page.

Reply:
I feel like we have lost the info about the `headers` being `null` - I haven't verified this is what truly does happen though, so I'm trusting that it was documented correctly to begin with. 😅